### PR TITLE
Shellstage sample 

### PR DESCRIPTION
**Pull request by RahulMR42**
# Description

Shell stage sample for newly LAd Deply feature.

Fixes # (issues)

New samples # (demos / Illustrations )

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] Free Tier
- [x] Always Free
- [x] Paid 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Updated/Sorted AIO.md file
- [x] My changes generate no new warnings.
- [x] All the sensitive informations are masked or removed.
- [x] All the links mentioned are valid and tested.
- [x] Image names are seperated only with '-'.
- [x] Added instruction to clone only this sample and links to other common sample pages.

### CODE REVIEW

File: oci-deployment-examples/oci-deployment-shell-stages/README.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+```java
+ALL {resource.type = 'devopsdeploypipeline', resource.compartment.id = 'COMPARTMENT_OCID'}
+```
+
+- Create a dynamic policy with the below statements.
+
+```java
+Allow dynamic-group  <NAME OF THE DYNAMIC GROUP> to manage virtual-network-family in compartment <NAME OF THE COMPARTMENT> 
+Allow dynamic-group <NAME OF THE DYNAMIC GROUP>  to manage ons-topics in compartment <NAME OF THE COMPARTMENT>
+Allow dynamic-group <NAME OF THE DYNAMIC GROUP>  to manage all-artifacts in compartment <NAME OF THE COMPARTMENT>
```
Comment (saparaj) [inline oci-deployment-examples/oci-deployment-shell-stages/README.md:56]: We need to have policy for compute-containers as well. Either "to manage compute-container-family" or specific policies as mentioned in TC doc.
https://confluence.oci.oraclecorp.com/display/DLCDEP/TC+documentation+details+-+Managed+Shell

File: oci-deployment-examples/oci-deployment-shell-stages/README.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+```
+
+### OCI DevOps setup
+
+- Create an `OCI Notification topic` - https://docs.oracle.com/en-us/iaas/Content/Notification/Tasks/create-topic.htm#top
+- Create a DevOps project and associate it with the topic. Ensure to enable logs for the DevOps project.
+
+![](images/oci-devops-logs.png)
+
+- Create a new DevOps artifact of type `command spec`
```
Comment (saparaj) [inline oci-deployment-examples/oci-deployment-shell-stages/README.md:66]: We are going to rename this type to "command specification" and there are UI changes related to rename too (from managed shell to shell), is this going to be used only for LA or GA?

File: oci-deployment-examples/oci-deployment-shell-stages/README.md (hunk 1 above)
Comment (RahulMR42) [reply oci-deployment-examples/oci-deployment-shell-stages/README.md]: thank you for the review ,
the policy  statement is added

File: oci-deployment-examples/oci-deployment-shell-stages/README.md (hunk 2 above)
Comment (RahulMR42) [reply oci-deployment-examples/oci-deployment-shell-stages/README.md:67]: Yes,its specific to LA ,we shall update after the GA.

**Review by saparaj — APPROVED** [review]

**Pull request merged**